app/src/constants/i18n.ts

```diff
@@ -341,8 +341,10 @@ export const SurveyAnimalsI18N = {
     'The species or taxon of the animal. If the species is unknown, select the lowest-ranking known taxon, such as the genus or family.',
```

Comment:
Typo: 'Different species may different units and unit names' should read 'Different species *may* have...'

Reply:
fixed